opcua/server/address_space.py

```diff
@@ -63,6 +63,9 @@ def write(self, params, user=UserManager.User.Admin):
                 if not ua.ua_binary.test_bit(al.Value.Value, ua.AccessLevel.CurrentWrite) or not ua.ua_binary.test_bit(ual.Value.Value, ua.AccessLevel.CurrentWrite):
                     res.append(ua.StatusCode(ua.StatusCodes.BadUserAccessDenied))
                     continue
+            tz = pytz.timezone('Europe/Berlin')
+            writevalue.Value.ServerTimestamp = datetime.now(tz)
```

Comment:
`datetime.now(tz.utc)` would be better, since every time thing on server side is based on UTC. Clients shall provide any conversions between UTC and local time.

Reply:
Agreed.